database-plugins/widgets/Database-batchsource.json

```diff
@@ -73,7 +73,7 @@
       "properties": [
         {
           "widget-type": "textbox",
-          "label": "Reference Name",
+          "label": "Table Name",
           "name": "referenceName",
           "widget-attributes": {
             "placeholder": "Name used to identify this source for lineage"
```

Comment:
This also needs to be updated

Reply:
Let's change this to `Name used to identify this source for lineage. Typically, the name of the table/view`